onboarding/src/Components/EmployeeProfile/ImplementationFormsToSend.js

```diff
@@ -16,12 +16,23 @@ function ImplementationFormsToSend(props) {
 
     const sendPackage = function(packageId){
         assignEmployeeToPackage(props.showModal, props.userId, packageId);
+        props.setCount(props.count + 1);
     };
 
-
-    form_table.forEach(function (element, i){
-        forms.push(<FormsToSendTableRow key={ element.key } row={ element } handleChecked={ showHide } handleSendPackage={ sendPackage } />);
-    });
+    const formsToSend = form_table.filter(form => !form.users.includes(props.userId));
+    if(formsToSend.length !== 0) {
+        formsToSend.forEach(function (element, i){
+            forms.push(<FormsToSendTableRow key={ element.key } row={ element } handleChecked={ showHide } handleSendPackage={ sendPackage } />);
+        });
+    } else {
+        forms.push(        
+            <tr key={ 0 }>
+                <td style={{ columnSpan: 7 }}>Brak formularzy do wysłania</td>
+            </tr>
+        );
+    }
+    console.log(forms)
+    console.log(form_table)
```

Comment:
A lot of `console.log`s ;-)
I'll let myself to comment them on a branch `react_dev`